Source/AwsCommonRuntimeKit/http/HTTPHeader.swift

```diff
@@ -28,3 +28,15 @@ public class HTTPHeader: CStruct {
         }
     }
 }
+
+extension Array where Element == HTTPHeader {
+    func withCHeaders<Result>(allocator: Allocator,
+                              _ body: (OpaquePointer) -> Result) -> Result {
+        var cHeaders: OpaquePointer = aws_http_headers_new(allocator.rawValue)
```

Comment:
leak? don't we need an `aws_http_headers_release()` somewhere?

and if this is a leak, why didn't tests catch it?

Reply:
Thanks, that is a great catch. The tests didn't catch it because I forgot to pass tracing alocator to sign chunk call. I have fixed and created a corresponding fix in aws-c-auth.